docs/network_firewall.rst

```diff
@@ -486,6 +562,114 @@ message "The changes have been applied successfully". Once you've set up the
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The below guide assumes you are using a 3 NIC firewall such as the SG-3100.
+While the SG-3100 has an integrated switch, you may need to add a switch to
+the LAN interface if you use a different firewall.
+
+Set Up LAN
+''''''''''
+
+Although we setup the LAN interface during the Setup Wizard we need to make
+a few revisions. Navigate in the WebGUI to configure the LAN interface. Go to
+**Interfaces ▸ LAN**, and ensure the **Enable Interface** box is checked. Use
+these settings:
+
+-  IPv4 Configuration Type: Static IPv4
+-  IPv4 Address: ``10.20.2.1`` (Application Gateway IP)
+
+Make sure that the CIDR routing prefix is correct (``/24``). Leave everything else
+as the default. **Save** and **Apply Changes**.
+
+|3 NIC LAN Interface|
+
+Set Up OPT1
+'''''''''''
+
+Next, you will have to enable the OPT1 interface. Go to
+**Interfaces ▸ OPT1**, and check the box to **Enable Interface**. Use these
+settings:
+
+
+-  IPv4 Configuration Type: Static IPv4
+-  IPv4 Address: ``10.20.3.1`` (Monitor Gateway IP)
+
+Make sure that the CIDR routing prefix is correct (``/24``). Leave everything else
+as the default. **Save** and **Apply Changes**.
+
+|3 NIC Firewall OPT1 Interface|
+
+
+Use Screenshots of Firewall Configuration
+'''''''''''''''''''''''''''''''''''''''''
+
+Here are some example screenshots of a working pfSense firewall
+configuration. You will add the firewall rules until they match what is
+shown on the screenshots.
+
+First, we will configure IP and port aliases. Navigate to **Firewall ▸ Aliases**
+and you should see a screen with no currently defined IP aliases:
+
+|Blank IP Aliases|
+
+Next you will click **Add** to add each IP alias.
+You should leave the **Type** as **Host**.
+Make aliases for the following:
+
+- ``admin_workstation``: ``10.20.2.3``
+- ``app_server``: ``10.20.2.2``
+- ``external_dns_servers``: ``8.8.8.8, 8.8.4.4``
+- ``monitor_server``: ``10.20.3.2``
+- ``local_servers``: ``app_server, monitor_server``
+
+|3 NIC Firewall Alias|
+
+Click **Save** to add the alias.
+
+Keep adding aliases until the screenshot matches what is shown here:
+
+|3 NIC Firewall IP Aliases Pre Save|
+
+Finally, click **Apply Changes**. This will save your changes. You should see a
+message "The changes have been applied successfully":
+
+|3 NIC Firewall IP Aliases Post Save|
+
+Next click "Ports" for the port aliases, and add the following ports:
+
+- OSSEC: ``1514``
+- ossec_agent_auth: ``1515``
+
+Your configuration should match this screenshot:
+
+|Port Aliases|
+
+Next we will configure firewall rules for each interface. Navigate to
+**Firewall ▸ Rules** to add firewall rules for the LAN and OPT1
+interfaces.
+
+.. warning:: Be sure not to delete the Anti-Lockout Rule on the LAN interface.
+    Deleting this rule will lock you out of the pfSense WebGUI.
+
+Add or remove rules until they match the following screenshots by clicking **Add**
+to add a rule.
+
+**LAN interface:**
+
+|3 NIC Firewall LAN Rules|
```

Comment:
nit: you can move the admin workstation TCP */* rule under the deny from LAN to OPT1 which should prevent the workstation from reaching the server over any port that 22.